compiler/src/main/antlr/Cellmata.g4

```diff
@@ -62,7 +62,9 @@ type_ident : IDENT | type_spec ;
 type_spec
     : array_decl # typeArray
     | TYPE_BOOLEAN # typeBoolean
-    | TYPE_NUMBER # tyopeNumber
+    | TYPE_NUMBER # typeNumber
+    | STMT_NEIGHBOUR # typeNeighbour
```

Comment:
It a bit weird that we have `TYPE_` prefix on some but `STMT_` prefix on other, but that is probably unfixable.

Reply:
Agreed, maybe write an issue on it?